client/src/component/VideoCard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import EmbeddingVideo from "./EmbeddingVideo";
+import Vote from "./Vote";
+
+function VideoCard(props){
+   
+
+    return(
+        <section className="cards-grid">
+            { 
+            props.videosList.sort((a,b) => b.rating - a.rating).map(film=>{
```

Comment:
I like that you're thinking about sorting up here. Does the sort array method mutate the original array? What are the side effects of doing this?

https://doesitmutate.xyz/sort/

Reply:
Yes it mutates the original array , but I wasn't aware of the side effects as I thought we just need the descending order.and keeping the original order might not be necessary.I will look into it. thanks